### PR TITLE
Multiple targets and syslog

### PR DESCRIPTION
**Pull request by omegas27**
I still need to test it but if you want to take a look, that would be great :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,52 +7,72 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## Unreleased
 
+## v0.3.1
+
+### Changed
+- Update fluentd to v1.10.2
+- Add syslog support
+- Can send logs to multiple targets using the copy plugin
+- Add nore customization options
```
Comment (pipo02mix) [inline CHANGELOG.md:16]:
```suggestion
- Add more customization options
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,52 +7,72 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
...
+[Unreleased]: https://github.com/giantswarm/fluent-logshipping-app/compare/v0.3.1...master
+[v0.3.1]: https://github.com/giantswarm/fluent-logshipping-app/releases/tag/v0.3.1
+[v0.3.0]: https://github.com/giantswarm/fluent-logshipping-app/releases/tag/v0.3.0
+[v0.2.2]: https://github.com/giantswarm/fluent-logshipping-app/releases/tag/v0.2.2
+[v0.2.1]: https://github.com/giantswarm/fluent-logshipping-app/releases/tag/v0.2.1
+[v0.2.0]: https://github.com/giantswarm/fluent-logshipping-app/releases/tag/v0.2.0
+[v0.1.3]: https://github.com/giantswarm/fluent-logshipping-app/releases/tag/v0.1.3
+[v0.1.2]: https://github.com/giantswarm/fluent-logshipping-app/releases/tag/v0.1.2
+[v0.1.1]: https://github.com/giantswarm/fluent-logshipping-app/releases/tag/v0.1.1
+[v0.1.0]: https://github.com/giantswarm/fluent-logshipping-app/releases/tag/v0.1.0
```
Comment (pipo02mix) [inline CHANGELOG.md:78]: <3

File: helm/fluent-logshipping-app/templates/fluentbit/configmap.yaml [hunk 3]
```diff
@@ -86,17 +128,48 @@ data:
         Match           *
         Host            ${FLUENTD_HOST}
         Port            ${FLUENTD_PORT}
+        Retry_Limit     false              # Retry
+
+  outputs-azure-loganalytics.conf: |
```
Comment (pipo02mix) [inline helm/fluent-logshipping-app/templates/fluentbit/configmap.yaml:133]: Where do you have the `include` directive for this one?

Comment (omegas27) [reply helm/fluent-logshipping-app/templates/fluentbit/configmap.yaml:133]: I actually do not, as this is wip to see if removing fluentd is worth it or not :( Let me remove it for this PR

**Comment by omegas27** [comment]
@pipo02mix do you mind taking another look? I did more changes than I thought

File: helm/fluent-logshipping-app/templates/fluentd/configmap.yaml [hunk 4]
```diff
@@ -209,6 +123,8 @@ data:
       {{- if .Values.fluentd.aws.S3.enabled }}
       <store>
         @type                 s3
+        @log_level            {{ .Values.fluentd.logLevel }}
+        log_es_400_reason     true
```
Comment (pipo02mix) [inline helm/fluent-logshipping-app/templates/fluentd/configmap.yaml:127]: Why do we need this only for S3? googling it looks it issued for ElasticSearch

**Pull request merged**